modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/DLSizeLimitConfigurationProviderImpl.java

```diff
@@ -45,6 +47,28 @@ public Map<String, Long> getCompanyMimeTypeSizeLimit(long companyId) {
 			companyId);
 	}
 
+	@Override
+	public Object[] getCopyToFailInfo(
+		long companyId, long groupId, long size, Locale locale) {
+
+		long groupMaxSizeToCopy = getGroupMaxSizeToCopy(groupId);
```

Comment:
I think that this should be:
long company
long group

..

you are using first the group so should be near the use. (I think see https://github.com/brianchandotcom/liferay-portal/commit/0d676d7823f15240543f5db2868a7cbc7e77a6a1)